epimodel/exports/epidemics_org.py

```diff
@@ -286,22 +303,48 @@ def upload_export(dir_to_export, gs_prefix, gs_url, channel="test"):
         "-a",
         "public-read",
     ]
-    gs_prefix = gs_prefix.rstrip("/")
-    gs_url = gs_url.rstrip("/")
+    gs_prefix = config["gs_prefix"].rstrip("/")
+    gs_url = config["gs_url_prefix"].rstrip("/")
     exdir = Path(dir_to_export)
     assert exdir.is_dir()
 
-    assert_valid_json(exdir / MAIN_DATA_FILENAME)
+    uploaddir = Path(config["output_dir"]) / config["output_uploads"]
+    if not uploaddir.exists():
+        uploaddir.mkdir()
+    channeldir = uploaddir / channel
+
+    if channeldir.exists():
+        shutil.rmtree(channeldir)
+
+    shutil.copytree(exdir, channeldir)
+    log.info(f"Copied export to {channeldir} for uploading")
+
+    datafile = config["gs_datafile_name"]
+    # Backwards compatibility
+    old_filename = "data-CHANNEL-v4.json"
+    old_path = channeldir / old_filename
+    if old_path.exists():
+        old_path.rename(channeldir / datafile)
+
+    for json_file in channeldir.iterdir():
+        if json_file.suffix != ".json":
+            continue
+        try:
+            assert_valid_json(json_file, minify=True)
+        except Exception as e:
+            log.error(f"Error in JSON file {json_file}")
+            raise e
 
-    log.info(f"Uploading data folder {exdir} to {gs_prefix}/{exdir.parts[-1]} ...")
-    cmd = CMD + ["-Z", "-R", exdir, gs_prefix]
+    log.info(
+        f"Uploading data folder {channeldir} to {gs_prefix}/{channeldir.parts[-1]} ..."
+    )
+    cmd = CMD + ["-Z", "-R", channeldir, gs_prefix]
     log.debug(f"Running {cmd!r}")
     subprocess.run(cmd, check=True)
 
-    datafile = MAIN_DATA_FILENAME.replace("CHANNEL", channel)
     gs_tgt = f"{gs_prefix}/{datafile}"
```

Comment:
Missing `datafile = MAIN_DATA_FILENAME` or similar

Reply:
I moved `MAIN_DATA_FILENAME` to `config.yml` and set `datafile = config["gs_datafile_name"]` in L322.